semantic_segmentation/scripts/train.py

```diff
@@ -137,13 +138,13 @@ def train(model, train_ds, val_ds, optimizer, writer,
 
 
 if __name__ == '__main__':
-    unique_name=datetime.now().strftime("%Y_%m_%d-%H_%M_%S")
+    unique_name = datetime.now().strftime("%Y_%m_%d-%H_%M_%S")
     writer = SummaryWriter('runs/u2net/'+unique_name)
     parser = argparse.ArgumentParser()
     parser.add_argument('--datafolder', type=str, default="")
     args = parser.parse_args()
     model = U2NET_lite()
-    #optimizer = optim.Adam(model.parameters(), lr=0.005)
+    # optimizer = optim.Adam(model.parameters(), lr=0.005)
```

Comment:
parametrization -> later

Reply:
from command line or just generally?

Reply:
cmd or better - config file

let's try to use lightning params - they have some extra functionality for this

check it here:
https://pytorch-lightning.readthedocs.io/en/stable/common/hyperparameters.html

Reply:
okay, so cmd for now, then integration in next step